### PR TITLE
Python Workflow

### PR DESCRIPTION
**Pull request by phillip-wenig-frequenz**
Adds workflows for:
- tests in Python with nox
- tests in Rust
- building the Python documentation

### CODE REVIEW

File: Cargo.toml.jinja [hunk 1]
```diff
@@ -3,7 +3,7 @@ name = "frequenz-{{project_name}}"
 description = "{{project_description}}"
 version = "0.1.0"
 edition = "2021"
-authors = ["Frequenz Energy-as-a-Service GmbH"]
+authors = ["{{project_author}}"]
```
Comment (llucax) [inline Cargo.toml.jinja:6]: Nitpick: This looks unrelated to adding the python CI so it could be split in a different commit.

**Review by llucax — COMMENTED** [review]
I only gave this a quick glance, as most of the files are probably copy&paste from other places. I will have a closer look to the new contributing file later.

File: Cargo.toml.jinja (hunk 1 above)
Comment (tiyash-basu-frequenz) [reply Cargo.toml.jinja:6]: This raises another question - who do we say the author is? The individual(s), or the team, or the company? Do you already use a standard on the python side of things?

**Review by tiyash-basu-frequenz — COMMENTED** [review]
Looks good from what I can understand :)

**Comment by llucax** [comment]
> This raises another question - who do we say the author is? The individual(s), or the team, or the company? Do you already use a standard on the python side of things?

Yeah, we are using always the company for this, it is used for the copyright notice for example, so it should be the company for our projects.

**Review by llucax — COMMENTED** [review]
The readme LGTM :+1:

**Review by llucax — APPROVED** [review]
For me this is OK to merge. If you want to split the commits, then great, feel free to force-merge if that's the only change.

**Pull request merged**